ciao-cli/template.go

```diff
@@ -61,6 +70,40 @@ func findField(fieldPath []string, v reflect.Value) reflect.Value {
 	return f
 }
 
+func filterBySubstr(obj interface{}, field, val string) (retval interface{}) {
```

Comment:
@markdryan makes perfect sense. I will also add a few more generic filters based on what the strings suport today and with the same nomenclature. That may help in other areas also, specially event processing.